cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -105,7 +105,13 @@ install_docker(){
 			sudo -E apt-get -y install apt-transport-https ca-certificates software-properties-common
 			repo_url="https://download.docker.com/linux/ubuntu"
 			curl -fsSL "${repo_url}/gpg" | sudo apt-key add -
-			sudo -E add-apt-repository "deb [arch=${arch}] ${repo_url} $(lsb_release -cs) stable"
+			# There is not a docker repository for Ubuntu 20.04, that is the main reason why
```

Comment:
What worries me is that presumably at some point docker will add a 20.04 repo and we might not notice. But I guess this is the only option we have for now.

It would be great if we could create a daily CI test that checked https://download.docker.com/linux/ubuntu/dists/ and looked for `focal` so we'd get a warning ;)

Reply:
Hmm, a random idea... we *could* make this code "auto-detect" the new repo by getting it to try to `add-apt-repository` for `focal` and if that doesn't work, use `bionic`. You'd have to be careful due to the `set -e` of course and there is also the issue again of how we'd detect / notice that the new repo had appeared so we could remove the "auto-detect" code.